warp10/src/main/java/io/warp10/continuum/gts/MetadataTextComparator.java

```diff
@@ -1,5 +1,5 @@
 //
-//   Copyright 2018  SenX S.A.S.
+//   Copyright 2018 - 2021  SenX S.A.S.
```

Comment:
Prefer `2018-2021`, without spaces, to keep the same format as other files.

Reply:
Done